teps/0096-pipelines-v1-api.md

```diff
@@ -75,6 +76,11 @@ A deprecated API version must continue to be supported for a length of time base
 | Alpha       | 0 releases                    | 0 months                  |
 | Beta        | 9 months or 3 releases        | 9 months                  |
 
+Kubernetes also defines the following stability policy for metrics:
+- Alpha metrics must function for 0 releases total and 0 releases after their announced deprecation.
+- Stable metrics must function for 4 releases or 12 months, and 3 releases or 9 months after their announced deprecation.
+Tekton does not define stability policies for metrics.
```

Comment:
NIT:I don't see it as something we should call out. The proposal includes a statement saying "Tekton follows the Kubernetes stability policy for metrics"  which is sufficient and clear.

Reply:
good point, done!